huaweicloud/services/lb/resource_huaweicloud_lb_loadbalancer.go

```diff
@@ -362,13 +362,13 @@ func resourceLoadBalancerV2SecurityGroups(networkingClient *golangsdk.ServiceCli
 	if v, ok := d.GetOk("security_group_ids"); ok {
 		securityGroups := resourcePortSecurityGroupsV2(v.(*schema.Set))
```

Comment:
it looks strange, as we using v1 client, but the function name has **v2** suffix